framework/lib/theme/components/badge/index.ts

```diff
@@ -37,4 +37,21 @@ export const Badge: ComponentSingleStyleConfig = {
       }
     },
   },
+  sizes: {
+    '2xs': {
+      fontSize: '2xs',
+    },
+    xs: {
+      fontSize: 'xs',
+    },
+    sm: {
+      fontSize: 'sm',
+    },
+    md: {
+      fontSize: 'md',
+    },
+  },
+  defaultProps: {
+    size: 'xs',
+  },
```

Comment:
It seems like at some parts of the mediatool codebase size Badge is used with size="sm", even tho it doesn't make a difference. But if we merge these changes then those badges will suddenly become one size bigger. So the defaultSize should probably be sm instead of xs and everything should move up one step: [2xs, xs, sm, md] => [xs, sm, md, lg].